authors/jeffrey_whewhetu.md

```diff
@@ -0,0 +1,4 @@
+Author: [Jeffrey Whewhetu]
+Title: [Open Source Software Engineer]
+Description: [I'm an open source software engineer who like contributing to open source projects and the communities around it. I also like solving complex problems.]
+Author Image: [assets/jeffrey-whewhetu.jpg]
```

Comment:
do you have org or company affiliation, linkedin or x to add?

Reply:
I don't have company affiliation but I came across the repo through algora but don't think it's okay to include them. As for my linkedin and X I included them